assets/PearInterests.txt

```diff
@@ -0,0 +1,15 @@
+Art
+Business
+Dance
+Design
+Fashion
+Fitness & Sports
+Food
+Humanities
+Music
+Photography
+Reading
+Sustainability
+Tech
+Travel
+TV & Film
```

Comment:
curious how fine-grain these interests should be since they're very broad right now

Reply:
Agreed, this is what design has right now but it is probably not set in stone.